src/APIsetup.js

```diff
@@ -0,0 +1,39 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api';
+
```

Comment:
[OPTIONAL]
- Amazing job! Your project works well.
- It's great that your project creates a new app for the API.
- If you want you could create only 1 app. Therefore, all users can submit their score to a general table and each one can compare with other users too. It is a suggestion, it is not a requirement.😉 
- In the end, both work well. 👍 🥇